src/github.com/getlantern/flashlight/pac.go

```diff
@@ -80,9 +80,19 @@ func genPACFile() {
 	formatter :=
 		`var bypassDomains = %s;
 		function FindProxyForURL(url, host) {
-			if (host == "localhost" || host == "127.0.0.1") {
+			if (isPlainHostName(host) // including localhost
+			|| shExpMatch(host, "*.local")) {
 				return "DIRECT";
 			}
+			// only checks plain IP addresses to avoid leaking domain name
+			if (/^[0-9.]+$/.test(host)) {
+				if (isInNet(host, "10.0.0.0", "255.0.0.0") ||
+				isInNet(host, "172.16.0.0",  "255.240.0.0") ||
+				isInNet(host, "192.168.0.0",  "255.255.0.0") ||
+				isInNet(host, "127.0.0.0", "255.255.255.0")) {
+					return "DIRECT";
+				}
+			}
```

Comment:
I guess I'm just thinking it's pretty unlikely users would ever enter those IPs, but I guess you're right that we might as well allow it in the off chance they do try them.

OK, merging!